server/eventlog_test.go

```diff
@@ -347,7 +350,66 @@ func TestParseSecureBootState(t *testing.T) {
 			t.Error("expected to see both WinProdPCA and ThirdPartyUEFI certs")
 		}
 	}
+}
+
+func TestParsingCELEventLog(t *testing.T) {
+	tpm := test.GetTPM(t)
+	defer client.CheckedClose(t, tpm)
+
+	coscel := &cel.CEL{}
+
+	testEvents := []struct {
+		cosNestedEventType cel.CosType
+		pcr                int
+		eventPayload       []byte
+	}{
+		{cel.ImageRefType, 13, []byte("docker.io/bazel/experimental/test:latest")},
```

Comment:
don't actually use PCR13 because `AppendEvent` has the side effect of extending real PCRs, and we sometimes test against real TPMs. Prefer Debug or Application

Reply:
If we expect COS events to always be in PCR13 and want to add that check in `parseCEL`, then we can tag this or similar tests such that we only run them on simulated TPMs.

Reply:
I changed to use test.DebugPCR. If we want use real TPM for testing we need to figure out a way to reset the debug pcr before/after the test. But seems like go-tpm hasn't implement this pct.PCRReset API yet.